vendor/github.com/ethereum/go-ethereum/whisper/whisperv5/whisper.go

```diff
@@ -540,16 +542,48 @@ func (w *Whisper) Stop() error {
 	return nil
 }
 
+func waitTrusted(peer *Peer) error {
+	timeout := time.After(time.Minute)
+	ticker := time.NewTicker(2 * time.Second)
+	defer ticker.Stop()
+	for {
+		select {
+		case <-timeout:
+			return p2p.DiscTooManyPeers
+		case <-ticker.C:
+			if peer.Trusted() {
+				return nil
+			}
+		}
+	}
+}
+
 // HandlePeer is called by the underlying P2P layer when the whisper sub-protocol
 // connection is negotiated.
 func (wh *Whisper) HandlePeer(peer *p2p.Peer, rw p2p.MsgReadWriter) error {
+	var peersOverflow bool
```

Comment:
Prefer more explicit `peersOverflow := false` opposite relying on default values here. Better matches to line 581 then.